### PR TITLE
Publish to PyPI

### PR DESCRIPTION
**Pull request by dirksammel**
This PR adds a first test of automatic publishing to PyPI (and some scripts to run as a service, needs to be improved and tested).

### CODE REVIEW

File: src/apel_plugin/publish.py [hunk 1]
```diff
@@ -31,15 +32,16 @@
 
 
 async def run(config, client):
+    logging.critical("START6")
```
Comment (QuantumDancer) [inline src/apel_plugin/publish.py:35]: Is this only for debugging or should this also be in production? If yes, maybe a more descriptive log message would be good.

File: src/apel_plugin/publish.py [hunk 2]
```diff
@@ -31,15 +32,16 @@
...
     report_interval = config["intervals"].getint("report_interval")
     time_db_path = config["paths"].get("time_db_path")
     publish_since = config["site"].get("publish_since")
     client_cert = config["authentication"].get("client_cert")
     client_key = config["authentication"].get("client_key")
-
+    logging.critical("START7")
```
Comment (QuantumDancer) [inline src/apel_plugin/publish.py:41]: Same here

File: src/apel_plugin/publish.py [hunk 3]
```diff
@@ -31,15 +32,16 @@
...
     token = get_token(config)
     logging.debug(token)
-
+    logging.critical("START8")
```
Comment (QuantumDancer) [inline src/apel_plugin/publish.py:44]: Same here

File: src/apel_plugin/publish.py [hunk 4]
```diff
@@ -117,18 +125,20 @@ def main():
         format=log_format,
         datefmt="%Y-%m-%d %H:%M:%S",
     )
+    logging.critical("START")
     logging.getLogger("asyncio").setLevel("WARNING")
     logging.getLogger("aiosqlite").setLevel("WARNING")
     logging.getLogger("urllib3").setLevel("WARNING")
-
+    logging.critical("START2")
```
Comment (QuantumDancer) [inline src/apel_plugin/publish.py:132]: Same here

File: src/apel_plugin/publish.py [hunk 5]
```diff
@@ -117,18 +125,20 @@ def main():
...
     auditor_ip = config["auditor"].get("auditor_ip")
     auditor_port = config["auditor"].getint("auditor_port")
-
+    logging.critical("START3")
```
Comment (QuantumDancer) [inline src/apel_plugin/publish.py:135]: Same here

File: src/apel_plugin/publish.py [hunk 6]
```diff
@@ -117,18 +125,20 @@ def main():
...
     builder = AuditorClientBuilder()
     builder = builder.address(auditor_ip, auditor_port)
     client = builder.build()
-
+    logging.critical("START4")
```
Comment (QuantumDancer) [inline src/apel_plugin/publish.py:139]: Same here

File: src/apel_plugin/publish.py [hunk 7]
```diff
@@ -117,18 +125,20 @@ def main():
...
     try:
+        logging.critical("START5")
```
Comment (QuantumDancer) [inline src/apel_plugin/publish.py:141]: Same here

File: scripts/install.sh [hunk 8]
```diff
@@ -0,0 +1,14 @@
...
+
+python -m venv venv
+source venv/bin/activate
+
+pip install --upgrade pip
+pip install -e .
+
+mkdir /etc/auditor-apel-plugin
+
+ln -s $PWD/apel_plugin.cfg /etc/auditor-apel-plugin/apel_plugin.cfg
```
Comment (QuantumDancer) [inline scripts/install.sh:11]: I noticed it here, but this comment is more general: Sometimes you use `"apel_plugin"` and sometimes `"auditor_apel_plugin"`. Should we be more consistent?

**Review by QuantumDancer — COMMENTED** [review]
LGTM, I only have two small comments (see inline)

File: src/apel_plugin/publish.py (hunk 1 above)
Comment (dirksammel) [reply src/apel_plugin/publish.py]: removed with latest commit

File: scripts/install.sh (hunk 8 above)
Comment (dirksammel) [reply scripts/install.sh:11]: Yes, the naming should be discussed at some point and will then be adjusted.

**Review by QuantumDancer — APPROVED** [review]

**Pull request merged**